tasks/timezone.yml

```diff
@@ -5,3 +5,15 @@
     dest: /etc/localtime
     state: link
     force: yes
+
+- name: clock file exist or not
+  shell: ls /etc/sysconfig/clock
```

Comment:
Use [stat module](https://docs.ansible.com/ansible/latest/modules/stat_module.html) to check if file exists